apps/web/public/static/locales/en/common.json

```diff
@@ -2006,6 +2010,7 @@
   "add_a_booking_question": "Add a question",
   "identifier": "Identifier",
   "duplicate_email": "Email is duplicate",
+  "enter_redirect_url_on_exit_description": "Enter a redirect URL on exit to redirect the attendee to a different page after the meeting ends.",
```

Comment:
I think we need to update this description - as I read this questions come to my mind:
1. `after the meeting ends.` - is the redirect happenining if I exit the meeting before it ends? Does `on exit` include that?
2. What happens if it is a 60 minute meeting and I stay past the time - will the redirect happen in the same tab as cal video or will the link will be opened in a new tab?

Reply:
This redirect would happen after user clicks on leave button

Reply:
Then I think we need to write that as the description, because otherwise "meeting ends" sounds like it may / may not include when the allocated time e.g. 60 minute meeting has passed.

I would write "Enter a redirect URL to which redirect the attendee once the attendee leaves the meeting." or something